Bukkit/src/main/java/info/preva1l/fadah/utils/StringUtils.java

```diff
@@ -151,4 +151,18 @@ public String capitalize(String str) {
         }
         return str.substring(0, 1).toUpperCase() + str.substring(1);
     }
+
+    /**
+     * Formats an item name. OAK_LOG -> Oak Log
+     * @param itemName item name
+     * @return formatted item name
+     */
+    public String formatItemName(String itemName) {
```

Comment:
for this, try with i18n item name first, then use this method if the i18n is null

Reply:
made myself, will merge and resolve merge conflicts